ember.md

```diff
@@ -357,3 +359,20 @@ Create a build definition for the app that will:
 
 ### 11. Checklist Violations
 `npm test` (in the build definition) will catch errors and reject build
+
+
+
+<a name="addons"></a>
+## Addons
+
+<a name="versioning"></a>
+### 9.1 Versioning
+
+1. We define `1.0.0` release as the first version ready to be used by the end user. It may not be feature complete or bug free, but it should bring some benefit to our customers. As soon as the package is ready to be used in production, it needs to be on version `1.0.0`.
+
+1. **Major** update should be any type of update that will break/change the way the addon is used. It could be an HTML structural update, a JS update, or a CSS change, that requires the end user to update the way the addon is used.
+
+1. **Minor** update should be an update that introduces a new feature and does not have any impact on how the addon is used.
+
+1. **Patch** update should be any type of bug fix that does not have any impact on how the addon is used.
```

Comment:
bug fix, documentation update, or small enhancement (e.g., performance) that does not add a new feature. also, should dependency updates go here? not ember updates, but smaller dependencies like addons